tealeaf/platform/PluginManager.h

```diff
@@ -40,6 +40,7 @@
 - (void) onPause;
```

Comment:
handleOpenURL is invoked when another app is requesting to handle resource.
Let app open from link be separate event.